packages/eas-cli/src/build/android/version.ts

```diff
@@ -2,13 +2,92 @@ import { ExpoConfig } from '@expo/config';
 import { AndroidConfig } from '@expo/config-plugins';
 import { Platform, Workflow } from '@expo/eas-build-job';
 import { BuildProfile } from '@expo/eas-json';
+import chalk from 'chalk';
+import fs from 'fs-extra';
 
+import Log from '../../log';
 import {
   getAppBuildGradleAsync,
   parseGradleCommand,
   resolveConfigValue,
 } from '../../project/android/gradleUtils';
 import { resolveWorkflowAsync } from '../../project/workflow';
+import { updateAppJsonConfigAsync } from '../utils/appJson';
+import { bumpShortVersionAsync, ensureStaticConfigExists } from '../utils/version';
+
+export enum BumpStrategy {
+  SHORT_VERSION,
+  VERSION_CODE,
+  NOOP,
+}
+export async function bumpVersionAsync({
+  bumpStrategy,
+  projectDir,
+  exp,
+}: {
+  projectDir: string;
+  exp: ExpoConfig;
+  bumpStrategy: BumpStrategy;
+}): Promise<void> {
+  if (bumpStrategy === BumpStrategy.NOOP) {
+    return;
+  }
+  ensureStaticConfigExists(projectDir);
+
+  const buildGradle = await getAppBuildGradleAsync(projectDir);
+  const isMultiFlavor =
+    buildGradle.android?.productFlavors ||
+    (buildGradle.android?.flavorDimensions ?? '').split(',').length > 1;
```

Comment:
```suggestion
    buildGradle.android?.flavorDimensions;
```
nit pick: without productFlavors flavorDimensions does not make sense